features/steps/step_selenium_account.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: UTF-8 -*-
+
+# @mark.steps
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+from behave import when, then
+
+
+@when('we are on Account settings Page')
+def step_impl(context):
+    context.settings_page.navigate()
+
+
+@then('we see filled account settings')
+def step_impl(context):
+    #assert len(context.settings_page.name()) > 0
+    for i in [context.settings_page.mention_name(),
+              context.settings_page.full_name(),
+              context.settings_page.email()]:
+        assert len(i) > 0
```

Comment:
I think it may use in future, as independent method which return just there is somethink in element/input or not, but, I agree that it not just one page object, there is method which we can use for any page in the project ,and I think we must to decide where we will keep method like that.

Reply:
I guess it should be function like `check_if_exist` and use like parameters str representation of "what we need to check" and return True/False. What about it?

Reply:
That sounds good, I think we need to add this task to our next split.

Reply:
I've added this method now and update with master, please check if is ok?